toolchain/sem_ir/typed_insts.h

```diff
@@ -354,6 +372,7 @@ struct BranchWithArg {
   InstId arg_id;
 };
 
+// A builtin instruction. These are special and listed in `builtin_kind.def`.
```

Comment:
I'm a little confused by your question, builtin_kind.def doesn't contain any functions (right now, at least). Perhaps you opened up builtin_function_kind.def instead?

Reply:
Maybe it'd be worth mentioning here that these are all builtin *types* specifically, other than `<error>` which can also be used as a non-type value.

Reply:
I've added an example. I really don't want to get into documenting what's already documented in builtin_kind.def though: to me, it's asking for skew because this doesn't own the contents of the enum.